consumption.tex

```diff
@@ -1,10 +1,20 @@
 \section{Consumption}
 
+Over the last 5 years there has been an explosion of innovation through new cryptocurrencies. Unfortunately, it is very hard for a new cryptocurrency to gain traction. Without traction, no market depth ensues, a cryptocurrency will have difficulty getting listed in any exchange. But without being listed in any exchange, a cryptocurrency cannot gain sufficient traction. The cryptocurrency can only be sold over-the-counter (OTC) by its miners or stakeholders, but that market is very limited.
+
+This chicken-and-egg situation presents the need for a solution that circumvents exchanges and allows users to acquire the cryptocurrency in a usable manner, in order for it to gain traction. We recommend utilizing proof-of-burn in order to allow users to obtain capital on a new cryptocurrency, automatically and without trusted third parties, by burning a legacy cryptocurrency.
```

Comment:
recommend -> propose

We don't recommend here, we make a proposal that each cryptocurrency can assess on their own to see if it's suitable for them.